driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/ClassFixtures.scala

```diff
@@ -0,0 +1,75 @@
+package org.virtuslab.ideprobe
+
+import java.nio.file.Path
+
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.virtuslab.ideprobe.ide.intellij.InstalledIntelliJ
+import org.virtuslab.ideprobe.ide.intellij.RunningIde
+
+trait RunningIntellijPerSuite {
+
+  private var workspace: Path = _
+  private var installed: InstalledIntelliJ = _
+  private var running: RunningIde = _
+  private var runningIntelliJFixture: RunningIntelliJFixture = _
```

Comment:
maybe lazy vals ?

Reply:
These are meant to be static later, kind of annoying but @BeforeClass and @AfterClass must be static, so the trait will be mixed in into an object. I don't think it is safe to allow single initialization in such case.